charts/rstudio-connect/README.md

```diff
@@ -142,6 +142,7 @@ The Helm `config` values are converted into the `rstudio-connect.gcfg` service c
 | launcher.templateValues | object | `{"job":{"annotations":{},"labels":{}},"pod":{"affinity":{},"annotations":{},"command":[],"containerSecurityContext":{},"defaultSecurityContext":{},"env":[],"extraContainers":[],"imagePullPolicy":"","imagePullSecrets":[],"initContainers":[],"labels":{},"nodeSelector":{},"priorityClassName":"","securityContext":{},"serviceAccountName":"","tolerations":[],"volumeMounts":[],"volumes":[]},"service":{"annotations":{},"labels":{},"type":"ClusterIP"}}` | Values to pass along to the Posit Connect session templating process |
 | launcher.templateValues.pod.command | list | `[]` | command for all pods. This is really not something we should expose and will be removed once we have a better option |
 | launcher.useTemplates | bool | `true` | Whether to use launcher templates when launching sessions. Defaults to true |
+| legacyPrometheus | bool | `false` | Whether to enable the legacy prometheusExporter INSTEAD OF the built-in product exporter. Takes precedence over    prometheusExporter.enabled |
```

Comment:
So this is more clear, but too many spaces.

```suggestion
| legacyPrometheus | bool | `false` | Whether to enable the legacy prometheusExporter INSTEAD OF the built-in product exporter. Takes precedence over prometheusExporter.enabled |
```